codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/middleware/MiddlewareBuilder.java

```diff
@@ -204,7 +210,10 @@ public void addDefaultMiddleware(GenerationContext context) {
 
         Middleware hostPrefix = (new Middleware.Builder())
                 .klass("Hearth::Middleware::HostPrefix")
-                .step(MiddlewareStackStep.INITIALIZE)
+                .step(MiddlewareStackStep.BUILD)
+                .relative(new Middleware.Relative(
+                        Middleware.Relative.Type.BEFORE, "Hearth::Middleware::Build")
```

Comment:
Is there a symbol for middleware here? Or do we have to pass "Hearth::Middleware::Build" as a string?

Reply:
We don't have symbols for any of these..... But we should.  I'll update all of our middleware to have symbol constants.